_includes/footer.html

```diff
@@ -42,10 +42,10 @@ <h3 class="c-vertical-navigation__title">Get involved</h3>
                   <a href="{{ 'join-us/' | relative_url }}">Become a Pirate!</a>
                 </li>
                 <li class="c-vertical-navigation__item">
-                  <a href="{{ '/feed.xml' | absolute_url }}">RSS / Atom feed</a>
+                  <a href="{{ '/feed.xml' | relative_url }}">RSS / Atom feed</a>
```

Comment:
Why shouldn't this be absolute?

Reply:
Is there a good reason for them not to be? The links are currently broken, though I get that may be resolved automatically when the new site is released

Reply:
Surely if you make them relative, when a user is on a sub page they are going to be directed to the wrong place?

Reply:
This doesn't seem to have been the case when I tested the change, but I'll go back and investigate further to see what's actually happening

Reply:
Ok, I've checked this again on the branch and yes, they're directed to the correct place. Not sure entirely how that's working though.